src/connector/src/sink/big_query.rs

```diff
@@ -255,6 +262,79 @@ impl BigQuerySink {
             ))),
         }
     }
+
+    fn map_field(rw_field: &Field) -> Result<TableFieldSchema> {
+        let tfs = match &rw_field.data_type {
+            DataType::Boolean => TableFieldSchema::bool(&rw_field.name),
+            DataType::Int16 | DataType::Int32 | DataType::Int64 | DataType::Serial => {
+                TableFieldSchema::integer(&rw_field.name)
+            }
+            DataType::Float32 => {
+                return Err(SinkError::BigQuery(anyhow::anyhow!(
+                    "Bigquery cannot support real"
+                )))
+            }
+            DataType::Float64 => TableFieldSchema::float(&rw_field.name),
+            DataType::Decimal => TableFieldSchema::numeric(&rw_field.name),
+            DataType::Date => TableFieldSchema::date(&rw_field.name),
+            DataType::Varchar => TableFieldSchema::string(&rw_field.name),
+            DataType::Time => TableFieldSchema::time(&rw_field.name),
+            DataType::Timestamp => TableFieldSchema::date_time(&rw_field.name),
+            DataType::Timestamptz => TableFieldSchema::timestamp(&rw_field.name),
+            DataType::Interval => {
```

Comment:
It seems to be supported, If it is not possible, please also set what is written below as not supported

Reply:
It should indeed be unsupported for now: https://cloud.google.com/bigquery/docs/schemas#standard_sql_data_types.
What needs to be added in the error message?

Reply:
I tested it before and it works, you can retest it, if it doesn't work, please disable `interval ` in the code elsewhere

Reply:
It does support it, but it is not a stable feature yet.
In addition, the sdk [FieldType](https://github.com/lquerel/gcp-bigquery-client/blob/cf3cd5c55bcd324ef24bf1f54af7f1a3da4aa729/src/model/field_type.rs#L5) have no `Internal`. It's used to construct `Table` that table().create() would use.
I think we can let it keep the unsupported error? Panic when validate failed.

Reply:
Forgot to change it in `get_string_and_check_support_from_datatype`?

Reply:
About `Interval`?
Unlike the case of `map_field`, which is used to create a BigQuery table, we can avoid actively using unstable type `Interval`; the return value of `get_string_and_check_support_from_datatype` is used to validate types of BigQuery table columns. If a column is indeed an `Interval` type, I don't think we have any reason to refuse?

Reply:
Got it, thanks for the explanation. Would you mind adding the explanations here into the code comment?